src/styles/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: var(--color-bg);
+  border: 1px solid var(--color-border);
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__heading {
+  margin-top: 40px;
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-txt-primary) ;
+}
+
+.card__code {
+  margin: 0 0 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-txt-secondary);
+}
+
+.card__review {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__review-paragraph {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-txt-primary);
+  margin: 0;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 24px 0 16px;
+}
+
+.card__price-heading {
+  margin: 0;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-txt-secondary);
+}
+
+.card__price-price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-txt-primary);
+}
+
+.card__link {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: var(--color-bg);
+  border: 1px solid transparent;
+  padding: 11px 0;
+  background-color: var(--color-link);
+  border-radius: 5px;
+  transition: color 0.2s linear;
+}
+
+.card__link:hover {
+  border-radius: 5px;
```

Comment:
The button already has this property, if the property is the same on hover you don't need to write it again

Reply:
removed